src/content/guides/multi-nginx/index.md

```diff
@@ -0,0 +1,81 @@
+
+Similarly, cloud load balancer created for each NGINX installation on AWS and Azure has to have unique host name associated with it. Host name suffix is common for all, and equals to the tenant cluster's base domain name. Prefix is configurable via `controller.service.subdomain` configuration property and defaults to `ingress`. In example configuration override for second internal NGINX IC App it is overriden to `ingress-internal`.
+
+For NGINX IC running on on-prem (KVM) tenant clusters there's no out-of-the-box `LoadBalancer` Service type support. Therefore NGINX Service type defaults to `NodePort`. For every NGINX IC App installation a set of unique http and https node ports ought to be assigned. The default NGINX http and https node ports are `30010` and `30011`. The example sets `31010` and `31011` as overrides for the internal NGINX.
+
+## Using weak ciphers for legacy clients
+
+In NGINX IC App v1.2.0 upgrade to ingress-nginx 0.27.1 was included. Among other things this upgrade brought in security improvements - SSL ciphers which are considered as weak got removed from the default configuration. Some of the older clients (like web browsers, http libraries in apps) could no longer establish secure connections with cluster services exposed via new NGINX, since these legacy clients only supported SSL ciphers that got removed.
+
+With single NGINX one had option of restoring weak SSL ciphers configuration, to support few services with older clients until the clients get upgraded. Problem with this approach, since SSL ciphers are global settings, was that changing default SSL ciphers back by restoring weak ciphers would apply to all Ingresses and service behind them, not just the one with old clients.
+
+With multiple NGINX ingress controller support, one can have separate NGINX installations with different SSL ciphers configuration, to localize but still make accessible services used by the legacy clients.
+
+Here is how this can be achieved:
+
+- Deploy one NGINX IC App using default settings
+- Deploy second internal NGINX IC App using following user configuration overrides
+
+  ```yaml
+  controller:
+    ingressClass: "nginx-weak"
+    service:
+      subdomain: "ingress-weak"
+  configmap:
+    ssl-ciphers: "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256"
+  ```
+
+For the second NGINX IC App installation ingress class and host name subdomain are customized for uniqueness. Additionally, default SSL ciphers are overriden to include weak ciphers for legacy clients.
```

Comment:
is the syntax or format for ssl-ciphers clear? (I am just asking)

Reply:
For the curious ones, upstream docs cover the syntax.